course-mw/sunbird-util/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraDACImpl.java

```diff
@@ -23,7 +22,59 @@
 import org.sunbird.common.responsecode.ResponseCode;
 
 public class CassandraDACImpl extends CassandraOperationImpl {
-    
+
+
+  @Override
+  public Response getBlobAsText(RequestContext requestContext, String keySpace, String table, Map<String, Object> filters, List<String> properties) {
+    Response response = new Response();
+    Session session = connectionManager.getSession(keySpace);
+
+    StringBuilder sb = new StringBuilder();
+    if (null != properties && !properties.isEmpty()) {
+      sb.append("select contentid, ");
```

Comment:
@pritha-tarento - the field "contentid" should not be hardcoded...

Reply:
Remove the contentid to address the comment